### DISCUSSION TITLE
Update rule with mqtt comand

### DISCUSSION THREAD

**Discussion by abhitheawesomecoder** [arendst/Tasmota#16087] [Q&A] [answered]
Is it possible to update a rule with mqtt command?

**Comment by sfromis** [comment, marked as answer]
Yes, any console command can be published to the [MQTT command](https://tasmota.github.io/docs/MQTT/#commands-over-mqtt) topic, including the Rule command to update a rule set.

**Comment by ascillato2** [comment]
**Example:**

Console command:
```JS
Rule1 ON Power1#Boot DO Var1 %value% ENDON
```

Assuming that your Tasmota Device Topic is `TEST`, the previous _console command_ can be sent by MQTT as:

Topic: `cmnd/TEST/Rule1`
Payload: `ON Power1#Boot DO Var1 %value% ENDON`
